include/fastdds/rtps/participant/RTPSParticipant.h

```diff
@@ -237,7 +237,7 @@ class RTPS_DllAPI RTPSParticipant
      * @brief Retrieves the built-in typelookup service manager.
      * @return
      */
-    fastdds::dds::builtin::TypeLookupManager* typelookup_manager() const;
+    fastdds::dds::builtin::TypeLookupManager& typelookup_manager() const;
```

Comment:
This API should be removed. TypeLookupService should not be exposed publicly.

Reply:
Removed in [9cf4645](https://github.com/eProsima/Fast-DDS/pull/4126/commits/9cf46451cbc0cc26f54f25c7979ada4904c22109)